setup.py

```diff
@@ -135,7 +144,14 @@ def get_extensions():
             include_dirs=tests_include_dirs,
             define_macros=define_macros,
             extra_compile_args=extra_compile_args,
-        )
+        ),
+        extension(
```

Comment:
I think one key decision to make is whether we want to cover building a "no-python" library here, too.
If we do, building this in Python setup.py (rather than with cmake) is a bit dubious. If we want this, I think we might want to refine this to not build a python module ([`cpp_extension.load`](https://pytorch.org/docs/stable/cpp_extension.html#torch.utils.cpp_extension.load) has a is_python_module flag, but the `*Extension` not.

If not, we might fold the custom ops into the extension module.

Reply:
This makes a lot of sense, and I imagine that one would want to run those models in C++ as well on torchscript.

One proposal: we can keep this approach as is for this PR, but we open a new issue discussing this?